Sources/SwiftTUI/View/Views/Stacks/ZStack.swift

```diff
@@ -0,0 +1,116 @@
+import Foundation
+
+public struct ZStack<Content: View>: View, Primitive, LayoutRoot {
+    public let content: Content
+    let alignment: Alignment
+
+    public init(alignment: Alignment = .center, @ViewBuilder _ content: () -> Content) {
+        self.content = content()
+        self.alignment = alignment
+    }
+
+    init(content: Content, alignment: Alignment = .center) {
+        self.content = content
+        self.alignment = alignment
+    }
+
+    static var size: Int? { 1 }
+
+    func loadData(node: Node) {
+        for i in 0 ..< node.children[0].size {
+            (node.control as! ZStackControl).addSubview(node.children[0].control(at: i), at: i)
+        }
+    }
+
+    func buildNode(_ node: Node) {
+        node.addNode(at: 0, Node(nodeBuilder: content.nodeBuilder))
+        node.control = ZStackControl(alignment: alignment)
+    }
+
+    func updateNode(_ node: Node) {
+        node.nodeBuilder = self
+        node.children[0].update(using: content.nodeBuilder)
+        let control = node.control as! ZStackControl
+        control.alignment = alignment
+    }
+
+    func insertControl(at index: Int, node: Node) {
+        (node.control as! ZStackControl).addSubview(node.children[0].control(at: index), at: index)
+    }
+
+    func removeControl(at index: Int, node: Node) {
+        (node.control as! ZStackControl).removeSubview(at: index)
+    }
+
+}
+
+private class ZStackControl: Control {
+    var alignment: Alignment
+
+    init(alignment: Alignment) {
+        self.alignment = alignment
+    }
+
+    // MARK: - Layout
+
+    override func size(proposedSize: Size) -> Size {
+        var size: Size = .zero
+        for control in children.sorted(by: { $0.verticalFlexibility(width: proposedSize.width) < $1.verticalFlexibility(width: proposedSize.width) }) {
```

Comment:
I think we can remove the use of `verticalFlexibility` here and in the `layout` method.